### PR TITLE
Set up React, Basement, CircleCi, FeaturePeek

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Sets up React, Basement, CircleCi, FeaturePeek
- Adds ReadMe and PR template

### Type(s) of changes

- [x] New feature

### Motivation for PR

#8 

### Follow-up PR

- Adding Sanity and ensuring that FeaturePeek works in my next PR

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="icon" href="%PUBLIC_URL%/assets/images/pratt-favicon-32x32.png" />
```
Comment (mokaymokay) [inline public/index.html:8]: remove?

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (mokaymokay) [inline public/robots.txt:3]: why do we want to disallow crawlers? just wondering

**Comment by chrismekim** [comment]
@mokaymokay @limsohee1002 I don't think FeaturePeek will ever get pinged by CircleCI because this PR adds CircleCi! So in my follow up PR adding Sanity, I'll make sure FP is working too. :)

**Review by mokaymokay — APPROVED** [review]

File: public/robots.txt (hunk 2 above)
Comment (chrismekim) [reply public/robots.txt:3]: From what I'm reading, if you don't want to crawlers to search certain pages, you would add something like `Disallow: /page` and they won't crawl `website.com/page`! This is create react app boilerplate :)

Comment (mokaymokay) [reply public/robots.txt:3]: ah, misread! & didn't know it comes with CRA

**Review by limsohee1002 — APPROVED** [review]

**Pull request merged**